jaggr-core/pom.xml

```diff
@@ -459,15 +459,5 @@
       <artifactId>powermock-core</artifactId>
       <scope>test</scope>
     </dependency>
-    <dependency>
```

Comment:
Correct. I had to remove our use of PowerMockRunner because of the need to run with -noverify JVM arg on Java 8, although we still use PowerMock for some things like Whitebox.